WPMedia/fusion-news-theme-blocks
# Tmedia 660 prettier eslint

## Description

Use prettier 

## Jira Ticket
- [TMEDIA-660](https://arcpublishing.atlassian.net/browse/TMEDIA-660)

## Acceptance Criteria
- Prettier fix all necessary files
- Setup git step for formatting changes

## Test Steps

1. Checkout this branch `git checkout TMEDIA-660-prettier-eslint`
2. Run `npm run lint`. Run prettier `npm run format`. don't see any conflicts
3. Tests pass as before. (I updated the snapshots as there were styled-component differences.)

## Effect Of Changes
### Before
Inconsistent code style

### After
More consistent code style

## Dependencies or Side Effects
- Saw some irregular whitespace linting change on one file where I disabled it with eslint. This seemeed to be the result of using inline snapshot
- Updated eslint as there were some requirements for updating that from other prettier packages

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress. -> na
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added. -> update vs code in editor suggestions and documentation

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
